airbyte-webapp/src/core/resources/DbMigration.ts

```diff
@@ -0,0 +1,62 @@
+import BaseResource from "./BaseResource";
+import { MutateShape, ReadShape, Resource, SchemaDetail } from "rest-hooks";
+
+export interface DbMigrationInfoItem {
+  migrationType: string;
+  migrationVersion: string;
+  migrationDescription: string;
+  migrationState?: string;
+  migratedBy?: string;
+  migratedAt?: bigint;
+  migrationScript?: string;
+}
+
+export interface DbMigrationInfoResult {
+  migrations: Array<DbMigrationInfoItem>;
+}
+
+export interface DbMigrationMigrateResult {
+  initialVersion: string;
+  targetVersion?: string;
+  migrationsExecuted: number;
+  migrations: Array<DbMigrationInfoItem>;
+}
+
+export default class DbMigrationResource extends BaseResource {
```

Comment:
So migrations object will always return this exact shape? It won't be possible to paginate migrations in the future, right?

Reply:
Yes, migration objects will always be the same shape. However, we do want to support pagination in the future.